### PR TITLE
the issue with duplicated channels persists

### PR DESCRIPTION
**Pull request by keichwa**
remove uyuni version number
and small cleanups

# Description

Short summary of why you created this PR

# Target branches

Which product version does this PR apply to?

- Manager-4.0
-  Manager-4.1
- [x] Manager-4.2
- [x] Uyuni

# Links

Fixes #<insert issue or PR link, if any>

### CODE REVIEW

File: modules/upgrade/pages/proxy-uyuni.adoc [hunk 1]
```diff
@@ -23,16 +23,16 @@ opensuse_leap15_2-uyuni-client \
 uyuni-proxy-stable-leap-152
 ----
 . Fully synchronize all channels with [command]``spacewalk-repo-sync``.
-    In case of already defined repository URLs, continue with <<uyuni-202007-channeldupes>>.
+  In case of already defined repository URLs, continue with <<uyuni-202007-channeldupes>>.
 
 
 
 [[uyuni-202007-channeldupes]]
```
Comment (juliogonzalez) [inline modules/upgrade/pages/proxy-uyuni.adoc:30]: There's still this 202007 thing here, right?

File: modules/upgrade/pages/proxy-uyuni.adoc [hunk 2]
```diff
@@ -23,16 +23,16 @@ opensuse_leap15_2-uyuni-client \
...
-== Uyuni 2020.07: Troubleshooting with Duplicated Repository URLs
+== Troubleshooting with Duplicated Repository URLs
 
 // https://github.com/SUSE/spacewalk/issues/12008
 
-If you want to add two or more repositories with the same URL (client tools for openSUSE Leap 15.0, 15.1, and 15.2) in {productname} 2020.07 with [command]``spacewalk-common-channels``, you will see an error such as:
+If you want to add two or more repositories with the same URL (client tools for openSUSE Leap 15.0, 15.1, and 15.2) in {productname} with [command]``spacewalk-common-channels``, you will see an error such as:
```
Comment (juliogonzalez) [inline modules/upgrade/pages/proxy-uyuni.adoc:35]: I see a small problem about this. the Leap channels on SUSE Manager should NOT be used from `spacewalk-common-channels` under any circumstances.

And right now the only "support" OS that uses `spacewalk-common-channels` on SUSE Manager is Debian, and AFAIK it's not affected by this.

Should we maybe have this for Uyuni only?

File: modules/upgrade/pages/proxy-uyuni.adoc [hunk 3]
```diff
@@ -23,16 +23,16 @@ opensuse_leap15_2-uyuni-client \
...
 ----
 . Fully synchronize all channels with [command]``spacewalk-repo-sync``.
-    In case of already defined repository URLs, continue with <<uyuni-202007-channeldupes>>.
+  In case of already defined repository URLs, continue with <<uyuni-202007-channeldupes>>.
 
 
 
 [[uyuni-202007-channeldupes]]
-== Uyuni 2020.07: Troubleshooting with Duplicated Repository URLs
+== Troubleshooting with Duplicated Repository URLs
```
Comment (Loquacity) [inline modules/upgrade/pages/proxy-uyuni.adoc:31]:
```suggestion
== Troubleshooting Duplicate Repository URLs
```

File: modules/upgrade/pages/proxy-uyuni.adoc (hunk 2 above)
Comment (Loquacity) [inline modules/upgrade/pages/proxy-uyuni.adoc:35]:
```suggestion
If you add two or more repositories with the same URL (client tools for openSUSE Leap 15.0, 15.1, and 15.2) in {productname} with [command]``spacewalk-common-channels``, an error like this is shown:
```

File: modules/upgrade/pages/proxy-uyuni.adoc [hunk 4]
```diff
@@ -47,15 +47,15 @@ Work around with assigning the 15.1 repository to the 15.2 channel:
 
 . In the upper right corner, click btn:[Manage Channel].
 
-. Click the [guimenu]``Repositories`` tab, and select [systemitem]``External - Uyuni Client Tools for openSUSE Leap 15.1 (x86_64)``
+. Click the [guimenu]``Repositories`` tab, and select [systemitem]``External - Uyuni Client Tools for openSUSE Leap 15.1 (x86_64)``.
```
Comment (Loquacity) [inline modules/upgrade/pages/proxy-uyuni.adoc:50]:
```suggestion
. Navigate to the [guimenu]``Repositories`` tab, and select [systemitem]``External - Uyuni Client Tools for openSUSE Leap 15.1 (x86_64)``.
```

File: modules/upgrade/pages/proxy-uyuni.adoc [hunk 5]
```diff
@@ -47,15 +47,15 @@ Work around with assigning the 15.1 repository to the 15.2 channel:
...
 
 . Click btn:[Update Repositories].
 
 . Navigate to menu:Repositories[Sync] subtab, and click btn:[Sync Now].
 
 . Do the same with [systemitem]``Uyuni Proxy Stable for openSUSE Leap 15.2 (x86_64)`` and [systemitem]``External - Uyuni Proxy Stable for openSUSE Leap 15.1 (x86_64)``.
 
-If you unfold the [systemitem]``openSUSE Leap 15.2 (x86_64)``, you will see all child channels propulated with packages.
+If you unfold the [systemitem]``openSUSE Leap 15.2 (x86_64)`` channel, you will see all child channels populated with packages.
```
Comment (Loquacity) [inline modules/upgrade/pages/proxy-uyuni.adoc:58]:
```suggestion
Expand the [systemitem]``openSUSE Leap 15.2 (x86_64)`` channel to see the child channels populated with packages.
```

File: modules/upgrade/pages/proxy-uyuni.adoc [hunk 6]
```diff
@@ -88,7 +88,7 @@ spacewalk-proxy stop
 . Click btn:[Next], and [guimenu]``Confirm Software Channel Change`` with btn:[Confirm].
 . Click menu:Software[Packages > Upgrade], and select all the packages to be updated on the proxy, and then apply the selection.
 . Re-register with the reactivation key using the [command]``rhnreg_ks`` command-line utility.
-    The system will be re-registered with the same id, history, and groups.
+  The system will be re-registered with the same id, history, and groups.
```
Comment (Loquacity) [inline modules/upgrade/pages/proxy-uyuni.adoc:91]:
```suggestion
  The system is re-registered with the same ID, history, and groups.
```

**Review by Loquacity — CHANGES_REQUESTED** [review]

File: modules/upgrade/pages/proxy-uyuni.adoc (hunk 1 above)
Comment (keichwa) [reply modules/upgrade/pages/proxy-uyuni.adoc:30]: yes, I kept it that way. It is just a label or ID.  I usually change those things only if we do a general cleanup (of if I think it is too confusing).

File: modules/upgrade/pages/proxy-uyuni.adoc (hunk 2 above)
Comment (keichwa) [reply modules/upgrade/pages/proxy-uyuni.adoc:35]: ok, thanks for the background info.  I think thus far we publish this in the Uyuni context only (but we should be prepared).  I think I must read again what we say about using Leap with suma.

Comment (keichwa) [reply modules/upgrade/pages/proxy-uyuni.adoc]: in the client config, Lana put in nice ifdefs.  I think everything is fine and I can merge this one.

**Comment by keichwa** [comment]
I need your ticks, though ;)

**Review by Loquacity — APPROVED** [review]

**Pull request merged**